hal/linux/configs/cocos_defconfig

```diff
@@ -7,6 +7,7 @@ BR2_TARGET_GENERIC_ISSUE="Welcome to Cocos"
 BR2_PACKAGE_DHCP=y
 BR2_PACKAGE_DHCP_CLIENT=y
 BR2_INIT_SYSTEMD=y
+BR2_SYSTEM_BIN_SH_BASH=y
```

Comment:
I have added `BR2_SYSTEM_BIN_SH_BASH` so that `bash` is the default script instead of BusyBox default `ash`.